javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/config/ConfigBuilder.java

```diff
@@ -16,53 +16,15 @@
 
 package io.opentelemetry.javaagent.tooling.config;
 
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_EXPORTER;
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_HYSTRIX_TAGS_ENABLED;
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_INTEGRATIONS_ENABLED;
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_KAFKA_CLIENT_PROPAGATION_ENABLED;
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_RUNTIME_CONTEXT_FIELD_INJECTION;
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_SQL_NORMALIZER_ENABLED;
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_TRACE_ENABLED;
-import static io.opentelemetry.instrumentation.api.config.Config.DEFAULT_TRACE_EXECUTORS_ALL;
 import static io.opentelemetry.instrumentation.api.config.Config.normalizePropertyName;
 
 import io.opentelemetry.instrumentation.api.config.Config;
-import java.util.Arrays;
-import java.util.Collections;
 import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
 import java.util.Map;
 import java.util.Properties;
-import java.util.function.Function;
-import org.checkerframework.checker.nullness.qual.Nullable;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public final class ConfigBuilder
```

Comment:
Is this class still needed/useful?

Reply:
I think it is: it allows us to reuse SDK `ConfigBuilder`'s environment variable and system property mechanism, it is a single place where we can specify the property reading order.